google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -470,4 +507,49 @@ private GrpcCallContext newCallContext(@Nullable Map<Option, ?> options, String
         metadataProvider.newExtraHeaders(resource, projectName));
     return context;
   }
+
+  public void shutdown() {
+    this.stub.close();
+    this.instanceStub.close();
+    this.databaseStub.close();
```

Comment:
The naming here is done in a previous PR by @pongad. If we want to change this I believe it would be better if we do it in a similar pull request, since there would be a lot of changes in the class.

Reply:
please add this to ur document for tracking

Reply:
done